### DISCUSSION TITLE
Questions about OKD license

### DISCUSSION THREAD

**Discussion by excellion75** [okd-project/okd#1171] [Q&A] [answered]
Hello.
I'm considering using OKD to distribute applications from the company I work for.
I would like to ask if OKD can be used for commercial purposes.

Also, I checked the following in the License item,
I wonder what "Some Components" is like here.

_"OKD is licensed under the Apache License, Version 2.0. **Some components may be licensed differently** - consult individual repositories for more."_

Thank you.

**Comment by vrutkovs** [comment, marked as answer]
An example might be Grafana - newer versions are GPL3-licenced, but afaik we include older, Apache2 version. So if Grafana (or some parts of it vendored in some other repo) is updated this component may use a different license
